### DISCUSSION TITLE
Issues with watchtower

### DISCUSSION THREAD

**Discussion by Koliera** [qdm12/gluetun#830] [Q&A]
Hi all, I've been using Gluetun on a single mega stack with all the usual *arr apps and it has been working great. The only thing has been that I'd like to use watchtower to keep up to the latest images, but it seems once Watchtower update one of the images in the stack or gluetun itself the whole thing becomes unstable and I have to stop the whole stack and start it up again. Has anyone tried something that could help?

**Comment by qdm12** [comment]
Yes. You should be able to update connected containers, but updating gluetun will disconnect containers. Subscribe to https://github.com/qdm12/deunhealth/issues/11 to know when this is done.
